fluid-music/src/vst2ToReaperObject.ts

```diff
@@ -0,0 +1,83 @@
+import { FluidPlugin, PluginType } from './plugin';
+import FluidIpcClient = require('./cybr/IpcClient');
+
+const rppp = require('rppp')
+const fluid = require('./cybr/index');
+
+/**
+ * Create a `ReaperVst` object from an existing plugin on the cybr instance.
+ *
+ * @param {FluidIpcClient} client A FluidIpcClient that is connected to a cybr instance
+ * @param {string} trackName - See documentation for fluid.audiotrack.select()
+ * @param {string} pluginName - See documentation for fluid.plugin.select()
+ * @param {string} pluginType
+ * @param {string} pluginId 
+ * 
+ * @returns {ReaperVst}
+ */
+export async function vst2ToReaperObject(client: FluidIpcClient, trackName: string, plugin: FluidPlugin, n: number, bpm: number) {
+
+  const cybrType = (plugin.pluginType === PluginType.unknown) ? undefined : plugin.pluginType;
+  const pluginName = plugin.pluginName;
+
+  const msg = [
+    fluid.audiotrack.select(trackName),
```

Comment:
This way of creating OSC messages will be deprecated at some point. I've been importing `const cybr = fluid.cybr;` at the top of the file, and then writing my helpers like this: `cybr.audiotrack.select('bass guitar'),`. This will ease the eventual separation of concerns between the `fluid-engine` and `cybr`.